tests/SegmentFieldTestModifier.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace SilverStripe\Forms\Tests;
+
+class SegmentFieldTestModifier extends AbstractSegmentFieldModifier
```

Comment:
No, it's a stub that was in the Test file - separated out. Should I move to Stub folder?